source/draw/gpu/opengl/OpenGLPacked.ooc

```diff
@@ -84,7 +85,7 @@ OpenGLPacked: abstract class extends GpuImage {
 					gpuMap add("texture0", image)
 			}
 		}
-		gpuMap useProgram(this, projection * view * model, textureTransform)
+		gpuMap useProgram(this, flat * projection * view * model, textureTransform)
```

Comment:
These matrices confirm that the assumption works for basic cropping and translation in normalized coordinates.
```
projection =
1.62414968, 0.00000000, 0.00000000, 0.00000000
0.00000000, -2.88737726, 0.00000000, 0.00000000
0.00000000, 0.00000000, 1.00000155, 0.02000002
0.00000000, 0.00000000, -1.00000000, 0.00000000

flat =
2.500000, 0.000000, 0.000000
0.000000, 2.500000, 0.000000
-0.781250, -0.925926, 1.000000	

flat * projection =
4.06037426, 0.00000000, 0.78125000, 0.00000000
0.00000000, -7.21844292, 0.92592591, 0.00000000
0.00000000, 0.00000000, 1.00000155, 0.02000002
0.00000000, 0.00000000, -1.00000000, 0.00000000
```